pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/offline/AbstractTableDataManager.java

```diff
@@ -30,78 +33,79 @@
 import java.util.concurrent.locks.ReadWriteLock;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
+import org.apache.helix.ZNRecord;
+import org.apache.helix.store.zk.ZkHelixPropertyStore;
 import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 
-// TODO: pass a reference to Helix property store during initialization. Both OFFLINE and REALTIME use case need it.
 public abstract class AbstractTableDataManager implements TableDataManager {
-  protected final List<String> _activeSegments = new ArrayList<String>();
-  protected final List<String> _loadingSegments = new ArrayList<String>();
+  protected final List<String> _activeSegments = new ArrayList<>();
```

Comment:
I believe _activeSegments and _loadingSegments are not used at all. Can be removed.

Reply:
They are useless but used lol. Removed.